.github/pull_request_template.md

```diff
@@ -1,41 +1,51 @@
-<!--
-Thank you for contributing to TiDB! Please read TiDB's [CONTRIBUTING](https://github.com/pingcap/community/blob/master/CONTRIBUTING.md) document **BEFORE** filing this PR.
+<!-- Thank you for contributing to TiDB!
+
+PR Title Format:
+1. pkg [, pkg2, pkg3]: what's changed
+2. *: what's changed
+
 -->
 
-### What problem does this PR solve? <!--add issue link with summary if exists-->
+### What problem does this PR solve?
 
+**Issue Number**: close #xxx <!-- REMOVE this line if no issue to close -->
 
-### What is changed and how it works?
+**problem summary**:
 
+### what is changed and how it works?
 
-### Check List <!--REMOVE the items that are not applicable-->
+**Proposal**: [xxx](url) <!-- REMOVE this line if not applicable -->
 
-Tests <!-- At least one of them must be included. -->
+**What's Changed**:
+
+**How it Works**:
 
- - Unit test
- - Integration test
- - Manual test (add detailed scripts or steps below)
- - No code
+### Related changes
 
-Code changes
+- PR to update `pingcap/docs` or `pingcap/docs-cn`:
+- PR to update `pingcap/tidb-ansible`:
+- Need to cherry-pick to the release branch
+
+### Check List <!--REMOVE the items that are not applicable-->
+
+Tests <!-- At least one of them must be included. -->
 
- - Has exported function/method change
- - Has exported variable/fields change
- - Has interface methods change
- - Has persistent data change
+- Unit test
+- Integration test
+- Manual test (add detailed scripts or steps below)
+- No code
 
 Side effects
 
- - Possible performance regression
- - Increased code complexity
- - Breaking backward compatibility
+- Performance regression
+    - Consumes more CPU
+    - Consumes more MEM
+- Breaking backward compatibility
 
-Related changes
+### Release note <!-- bugfixes or new feature need a release note -->
 
- - Need to cherry-pick to the release branch
- - Need to update the documentation
- - Need to update the `tidb-ansible` repository
+Please choose at least one of them to write the release note:
 
-Release note
+- **Release Note(in English)**:
 
- - Write release note for bug-fix or new feature.
+- **Release Note(in Chinese)**:
```

Comment:
I think it's not appropriate.

Reply:
+1

Reply:
I think we need more comment for `Release Note`, this may confuse contributor.